ansible/roles/logstash-job-openshift/defaults/main.yaml

```diff
@@ -14,3 +14,7 @@ job_name: logstash-job-1
 configs_dir: "{{ role_path }}/files/"
 # Define certs_dir to add certificates to the job
 # certs_dir: "{{ role_path }}/files/"
+
+
+query_parameter_field: ci_job.name
```

Comment:
please remove this parameter and hard code ci_job.name field

Reply:
done